js/appInit.ts

```diff
@@ -42,6 +42,11 @@ function registerTableView() {
     window["ActionBlock"] = ActionBlock;
 }
 
+/** Override configuration on test pages */
+function overrideConfig() {
+    overrideConfiguration(window["pokerConfig"] || {});
```

Comment:
Are you able export `overrideConfiguration` and not create custom function `overrideConfig()`

Reply:
I can't pass parameter to this function from test project.

Reply:
What prevent passing parameter? Since you export function, it should just work.